Flank/flank
# fix: Fix dry run

Fixes #1668 

## Test Plan
> How do we know the code works?

1. run `./gradlew flankFullRun`
2. start any flank run but add `--dry` flag a the end
3. dry run should proceed normally, without any errors
